store/postgres/src/relational.rs

```diff
@@ -170,8 +168,43 @@ impl Layout {
         let mut tables = Vec::new();
         let mut enums = EnumMap::new();
 
-        for defn in &document.definitions {
-            match defn {
+        if create_proof_of_indexing {
+            let poi_table = Table {
+                object: "poi_artifact".to_owned(),
+                name: SqlName::from("poi_artifact"),
+
+                qualified_name: SqlName::qualified_name(&schema, &SqlName::from("poi_artifact")),
+
+                columns: vec![
+                    Column {
+                        name: SqlName::from("artifact"),
+                        field: "artifact".to_owned(),
+                        field_type: q::Type::NonNullType(Box::new(q::Type::NamedType(
+                            "String".to_owned(),
+                        ))),
+                        column_type: ColumnType::String,
+                        fulltext_fields: None,
+                    },
+                    Column {
+                        name: SqlName::from(PRIMARY_KEY_COLUMN),
+                        field: PRIMARY_KEY_COLUMN.to_owned(),
+                        field_type: q::Type::NonNullType(Box::new(q::Type::NamedType(
+                            "String".to_owned(),
+                        ))),
+                        column_type: ColumnType::String,
+                        fulltext_fields: None,
+                    },
+                ],
+                /// The position of this table in all the tables for this layout; this
+                /// is really only needed for the tests to make the names of indexes
+                /// predictable
+                position: tables.len() as u32,
+            };
```

Comment:
How fun, a manually constructed `Table` structure :laughing:

Reply:
Was there a better way to do this? I didn't want to create a schema type and pass that to `Table::new`.

Reply:
No, that's perfectly fine and probably the best you can do here w/o huge overhead